.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Recheck the structure, I saw `OpenQuestions` is same level as these files, create another folder call `practice-one` put everything in that